docs/datastorage/capabilities.md

```diff
@@ -70,12 +72,12 @@ It is strongly suggested that direct checks in code are used to test for capabil
 Attaching Capabilities
 ----------------------
 
-As mentioned, attaching capabilities to entities and itemstacks can be done through the `AttachCapabilityEvent`. This event has 3 sub-events, for extra granularity:
+As mentioned, attaching capabilities to entities and itemstacks can be done through the `AttachCapabilityEvent`. This event has 4 sub-events, for extra granularity:
 
-* `AttachCapabilityEvent.Entity`: Fires only for entities.
-* `AttachCapabilityEvent.TileEntity`: Fires only for tile entities.
-* `AttachCapabilityEvent.Item`: Fires only for item stacks.
-* `AttachCapabilityEvent.World`: Fires only for worlds.
+* `AttachCapabilityEvent<Entity>`: Fires only for entities.
+* `AttachCapabilityEvent<TileEntity`: Fires only for tile entities.
+* `AttachCapabilityEvent<Item>`: Fires only for item stacks.
+* `AttachCapabilityEvent<World>`: Fires only for worlds.
```

Comment:
While you are at it, I suggest you reword this part, since it's not really using sub-events anymore, but the new generic event system.